pkg/dataflatten/jsonl.go

```diff
@@ -0,0 +1,36 @@
+package dataflatten
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+)
+
+func JsonlFile(file string, opts ...FlattenOpts) ([]Row, error) {
+	f, err := os.Open(file)
+	if err != nil {
+		return nil, err
+	}
+	return Jsonl(bufio.NewReader(f), opts...)
```

Comment:
Why `bufio.NewReader(f)` here?

Reply:
I thought that bufio would do some magic under the the hood to make reading the file more efficient. I ran some bench marks using a file with millions of lines and there was no noticeable difference. Dropped bufio, now just passing file.